apps/coa/behaviours/save-document.js

```diff
@@ -37,7 +37,7 @@ module.exports = (documentCategory, fieldName) => superclass => class extends su
       if (invalidSize) {
         return validationErrorFunc('maxFileSize');
       } else if (invalidMimetype) {
-        return validationErrorFunc('fileType', ['JPG, JPEG, PNG or PDF']);
```

Comment:
`JPG, JPEG, PNG or PDF` This has been moved to translations as it is language specific.